src/lib/y2firewall/clients/installation_finish.rb

```diff
@@ -45,15 +52,20 @@ def modes
 
       def write
         Service.Enable("sshd") if @settings.enable_sshd
-        @firewalld.enable! if @settings.enable_firewall
+
+        return true unless @settings.enable_firewall
```

Comment:
so if firewall is not enabled, we did not disable it?

Reply:
Well, this client is the only one that enables the service, make sense to disable it during installation?

Reply:
well, it depends if user click on disable firewall and product have it enabled by default, I expect it will be disabled.